### PR TITLE
feat: Service Management Area documentation

### PR DESCRIPTION
**Pull request by Jelledb**
This is still a Work in Progress while we wait for the Approval flows to come in.

### CODE REVIEW

**Comment by uleMeshcloud** [comment]
image

File: docs/marketplace.service-management-area.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+title: Service Management Area
+---
+
+> Please note that the following functionality is only available to meshStacks that have the **Service Economy** module activated.
+> Reach out to your Customer Success representative if you are interested in using this module.
+
+## Introduction
+
+The Service Management Area can be used by workspaces to offer their [Platform Services](./marketplace.index.md#platform-services) to the rest of the organization.
+For example, the Azure Team in your organization could set up a workspace to offer their Azure [Platform](./administration.platforms.md) from there to
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:12]: Here its Azure Team upper case, in the next line Network team mixed case. I am in favor of having it just lower case

File: docs/marketplace.service-management-area.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Anyone in the organization can offer their valuable services to the rest of the organization through the
+Service Management Area as long as it is approved by someone in the [Admin Area](./administration.index.md).
+
+The Service Management Area can at any time be opened up from the Workspace Control Plane by clicking the
+"Go to Services Management" button as depicted below.
+
+![Introduction](assets/service-management-area/introduction.png)
+
+If you would like to go back again to the regular Workspace view, click on "Go to Workspace Management".
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:24]: If we make Admin Area upper case we should also make Workspace View upper case

File: docs/marketplace.service-management-area.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Types of Platform Services
+
+The following three types of platform services can be created and offered in the Service Management Area:
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:28]: We should platform services always in lower case. Can you make that consistent please

File: docs/marketplace.service-management-area.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+2. [Building Blocks](./administration.building-blocks.md). Building Blocks are standardized extensions to cloud tenants that users
+   can book and roll out. These can be either rolled out using Terraform, or manually. Example use cases are an on-premise
+   connectivity to a cloud tenant.
+3. [Service Brokers](./administration.service-brokers.md). Service Brokers are self-hosted components that can execute
+   automation and provision workloads using the Open Service Broker API that can be booked by users in the meshStack.
+   As the automation is fully managed by someone it can provision and do anything that you would like.
+
+## Development & Testing
+
+By default, anything that you create in your Service Management Area will not be available to anyone else and requires approval.
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:43]:
```suggestion
By default, anything that you create in your Service Management Area will be available to you inside the workspace and not directly published in the Service Catalog.
```

File: docs/marketplace.service-management-area.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+However, for development & testing purposes, you can consume any of your created platform services mentioned above in
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:44]:
```suggestion
For development & testing purposes, you can consume any of your created platform services mentioned above in
```

File: docs/marketplace.service-management-area.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+**your own workspace**. They will be marked as "Private". This will give you an idea of the user experience and allows you
+to debug any potential issues with the Platform Configuration or Terraform code.
+
+![Private](./assets/service-management-area/private.png)
+
+
+## Publishing
+
+After you are done with the development & testing phase, you can go the control plane of your provided service in the Service Management Area and submit it for publication by clicking the "Submit for publishing" button.
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:53]:
```suggestion
After you are done with the development & testing phase, you can go the Control Plane of your provided service in the Service Management Area and submit it for publication by clicking the "Submit for publishing" button.
```

File: docs/marketplace.service-management-area.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Once you've addressed the identified issues, you can resubmit it for publishing.
+
+
+## Approval
+
+To review a service, go to "Workspaces Services" under "Marketplace" on the left side bar. The "Workspace Services" provides you an overview of all services which are submitted for publishing. 
+
+![Workspace Services](./assets/service-management-area/workspace-services.png)
+
+Upon selecting a service, you will be directed to its control plane. From there, you can either reject the publication request or approve it, making the service accessible to all workspaces within your organization.
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:71]:
```suggestion
Upon selecting a service, you will be directed to its Control Plane. From there, you can either reject the publication request or approve it, making the service accessible to all workspaces within your organization.
```

File: docs/marketplace.service-management-area.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+Once approved by your cloud foundation team, it will become accessible to all workspaces within the organization.
+If your publication request gets rejected, you can easily identify the reason by hovering over the rejected label or checking the rejection comment in the history.
+
+![Rejection](./assets/service-management-area/rejection-details.png)
+
+Once you've addressed the identified issues, you can resubmit it for publishing.
+
+
+## Approval
```
Comment (chewbeccy) [inline docs/marketplace.service-management-area.md:65]: I think this part belongs into the Admin Docs Part

**Review by chewbeccy — APPROVED** [review]
Found some minor things. Please have a look before merging

File: docs/marketplace.service-management-area.md (hunk 8 above)
Comment (Jelledb) [reply docs/marketplace.service-management-area.md:65]: I move it to another dedicated page.

**Pull request merged**